mmv1/templates/terraform/constants/resource_dns_resource_record_set.go.erb

```diff
@@ -11,9 +18,28 @@ func rrdatasDnsDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
 	return false
 }
 
-func ipv6AddressforDnsDiffSuppress(_, old, new string, _ *schema.ResourceData) bool {
-	oldIp := net.ParseIP(old)
-	newIp := net.ParseIP(new)
-
-	return oldIp.Equal(newIp)
+func ipv6AddressforDnsDiffSuppress(x, y []string, _ *schema.ResourceData) bool {
```

Comment:
Prefer other variable names than x and y. Maybe keep old, new like the previous function did?

Reply:
Sure will update accordingly